polaris-core/src/main/java/org/apache/polaris/core/entity/PolarisPrincipalSecrets.java

```diff
@@ -64,40 +73,78 @@ private String generateRandomHexString(int stringLength) {
     return sb.toString();
   }
 
+  private String hashSecret(String secret) {
+    return DigestUtils.sha256Hex(secret + ":" + secretSalt);
```

Comment:
It is probably advisable to use different salt values for each secret (for the same reason why different salt values are used for different users).

Reply:
I am actually not clear on why/if we need two secrets per user. 

If we turn out to only want to use one, then just having the one salt is fine. If we are really using both secrets, then probably having two salts for the two secrets is better.